hivemq4/base-image/Dockerfile

```diff
@@ -37,27 +36,35 @@ RUN set -x \
         && { command -v gpgconf && gpgconf --kill all || :; } \
         && chmod +x /usr/local/bin/gosu \
         && gosu nobody true \
-        && apt-get purge -y gpg dirmngr && rm -rf /var/lib/apt/lists/* \
-        && mkdir -p /docker-entrypoint.d
+        && apt-get purge -y gpg dirmngr && rm -rf /var/lib/apt/lists/*
+
+
+#Additional build image to unpack the zip file and change the permissions without retaining large layers just for those operations
+FROM javabase as unpack
```

Comment:
why do we need javabase here?

Reply:
I guess the whole javabase is not needed if we have 1 unpack build stage and 1 final image build step

Reply:
simplified it to 2 build stages